source/reference/minio-mc/mc-sql.rst

```diff
@@ -145,7 +145,7 @@ Parameters
 .. mc-cmd:: --json-input
    
 
-   *Optional* The data format for ``.json`` input objects. Specify the type of 
+   *Optional* The data format for ``.json`` or ``.ndjson`` input objects. Specify the type of 
```

Comment:
Should we use the `:optional:` syntax here?
You fixed the other file, but may not want to fix all of them in this file. Up to you.

Reply:
I contemplated but ended up not, as this file had _something_ at least.

It's not difficult, yeah might as well. I'll do it before merge, holding this overnight to see if there's any final eng input. 👍🏻